### DISCUSSION TITLE
Time step for simulation output data

### DISCUSSION THREAD

**Discussion by Abhinav-Kshirsagar** [pybamm-team/PyBaMM#3213] [Q&A]
Dear developer,
While I am solving the simulation for 1C (dis)charge using the DFN model and Casadi solver, I am unable to control the time at which my output results are logged.
Does dt_max only consider the time at which the solution is solved?
How can I control at which the output is logged?  On observing the time vector, I observe it is always having a time step of 60 secs.
Thanks in advance for your help.

**Comment by valentinsulzer** [comment]
You can change the period of any step or the whole experiment https://docs.pybamm.org/en/latest/source/examples/notebooks/getting_started/tutorial-5-run-experiments.html

**Reply by Abhinav-Kshirsagar** [reply]
Dear Mr. Sulzer,
Thank you for your quick repsonse. 
I tried the options stated in Tutorial 5.
I am getting the following AttributeError: module 'pybamm' has no attribute 'step'.
I am using version 23.4.1. 
Attached is my code for reference.
Could you please help me out with this?
Thank you.
[Unable_to_implement_step_size.ipynb.zip](https://github.com/pybamm-team/PyBaMM/files/12225753/Unable_to_implement_step_size.ipynb.zip)

**Reply by brosaplanella** [reply]
Step is only available from 23.5 onwards. If you upgrade your PyBaMM version it should work.
